asdf/tags/core/ndarray.py

```diff
@@ -244,13 +246,20 @@ def __init__(self, source, shape, dtype, offset, strides,
             self._make_array()
 
     def _make_array(self):
+        if self._block_data_ref is not None:
```

Comment:
A comment string something like
```
# Check if self.block.data has moved
```
or similar will help our future selves remember why this block of code is here.

Reply:
Added a comment